src/view/com/posts/FeedItem.tsx

```diff
@@ -130,18 +130,19 @@ let FeedItemInner = ({
     })
   }, [post, record, openComposer])
 
-  const outerStyles = [
-    styles.outer,
-    pal.view,
-    {
-      borderColor: pal.colors.border,
-      paddingBottom:
-        isThreadLastChild || (!isThreadChild && !isThreadParent)
-          ? 6
-          : undefined,
-    },
-    isThreadChild ? styles.outerSmallTop : undefined,
-  ]
+  const outerStyles = useStyle(React.useMemo(() => ({
+    bg: 'l0',
+    borderTopWidth: isThreadChild ? 0 : 1,
+    paddingLeft: 'm',
+    pr: 'l',
+    cursor: 'pointer',
+    overflow: 'hidden',
+    borderColor: 'l2',
+    paddingBottom:
+      isThreadLastChild || (!isThreadChild && !isThreadParent)
+        ? 's'
+        : undefined,
+  }), []))
```

Comment:
Didn't replace `Link`, so stuck with the style object. Easier to read like this and know what styles are being applied.